aderyn_core/src/context/meta_workspace.rs

```diff
@@ -0,0 +1,239 @@
+use crate::ast::*;
```

Comment:
What's the difference between workspace and meta_workspace? Why the separation?

Reply:
There is No difference. The file was getting too big so I thought I'll put it in a `meta` file :P